sql/moz-fx-data-shared-prod/static/country_codes_v1/data.csv

```diff
@@ -1,253 +1,253 @@
-Unknown Country Name,??,Unknown Region,Unknown Sub-region,Unknown Intermediate Region
\ No newline at end of file
+name,code,region_name,subregion_name,intermediate_region_name,pocket_available_on_newtab,mozilla_vpn_available
+Afghanistan,AF,Asia,Southern Asia,Unspecified,FALSE,FALSE
+Åland Islands,AX,Europe,Northern Europe,Unspecified,FALSE,FALSE
+Albania,AL,Europe,Southern Europe,Unspecified,FALSE,FALSE
+Algeria,DZ,Africa,Northern Africa,Unspecified,FALSE,FALSE
+American Samoa,AS,Oceania,Polynesia,Unspecified,FALSE,FALSE
+Andorra,AD,Europe,Southern Europe,Unspecified,FALSE,FALSE
+Angola,AO,Africa,Sub-Saharan Africa,Middle Africa,FALSE,FALSE
+Anguilla,AI,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE
+Antarctica,AQ,Unspecified,Unspecified,Unspecified,FALSE,FALSE
+Antigua and Barbuda,AG,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE
+Argentina,AR,Americas,Latin America and the Caribbean,South America,FALSE,FALSE
+Armenia,AM,Asia,Western Asia,Unspecified,FALSE,FALSE
+Aruba,AW,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE
+Australia,AU,Oceania,Australia and New Zealand,Unspecified,FALSE,FALSE
+Austria,AT,Europe,Western Europe,Unspecified,TRUE,FALSE
+Azerbaijan,AZ,Asia,Western Asia,Unspecified,FALSE,FALSE
+Bahamas,BS,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE
+Bahrain,BH,Asia,Western Asia,Unspecified,FALSE,FALSE
+Bangladesh,BD,Asia,Southern Asia,Unspecified,FALSE,FALSE
+Barbados,BB,Americas,Latin America and the Caribbean,Caribbean,FALSE,FALSE
+Belarus,BY,Europe,Eastern Europe,Unspecified,FALSE,FALSE
+Belgium,BE,Europe,Western Europe,Unspecified,TRUE,FALSE
+Belize,BZ,Americas,Latin America and the Caribbean,Central America,FALSE,FALSE
+Benin,BJ,Africa,Sub-Saharan Africa,Western Africa,FALSE,FALSE
+Bermuda,BM,Americas,Northern America,Unspecified,FALSE,FALSE
+Bhutan,BT,Asia,Southern Asia,Unspecified,FALSE,FALSE
+"Bolivia, Plurinational State of",BO,Americas,Latin America and the Caribbean,South America,FALSE,FALSE
+"Bonaire, Sint Eustatius and Saba",BQ,Americas,Latin America and the Caribbean,Unspecified,FALSE,FALSE
+Bosnia and Herzegovina,BA,Europe,Southern Europe,Unspecified,FALSE,FALSE
+Botswana,BW,Africa,Sub-Saharan Africa,Southern Africa,FALSE,FALSE
+Bouvet Island,BV,Americas,Latin America and the Caribbean,South America,FALSE,FALSE
+Brazil,BR,Americas,Latin America and the Caribbean,South America,FALSE,FALSE
+British Indian Ocean Territory,IO,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE
+Brunei Darussalam,BN,Asia,South-eastern Asia,Unspecified,FALSE,FALSE
+Bulgaria,BG,Europe,Eastern Europe,Unspecified,FALSE,FALSE
+Burkina Faso,BF,Africa,Sub-Saharan Africa,Western Africa,FALSE,FALSE
+Burundi,BI,Africa,Sub-Saharan Africa,Eastern Africa,FALSE,FALSE
+Cambodia,KH,Asia,South-eastern Asia,Unspecified,FALSE,FALSE
+Cameroon,CM,Africa,Sub-Saharan Africa,Middle Africa,FALSE,FALSE
+Canada,CA,Americas,Northern America,Unspecified,TRUE,TRUE
```

Comment:
VPN is also currently available in 11 additional countries (17 total):  Austria, Belgium, Finland, France, Germany, Ireland, Italy, Netherlands, Spain, Sweden, Switzerland

Reply:
Replied offline